spring-cloud-dataflow-server-core/src/main/resources/schemas/mysql/V1-skipper.sql

```diff
@@ -0,0 +1,269 @@
+create table if not exists hibernate_sequence (
```

Comment:
I know this is how it is setup.   Just seems odd that the skipper migration is in dataflow and not in skipper.

Reply:
Yeh it is a bit odd but it expects that SCDF "goes first" and if it does then it must provide the schemas.